### DISCUSSION TITLE
Argument to start / execute MWZ via Voyager Script

### DISCUSSION THREAD

**Discussion by rkas12** [mworion/MountWizzard4#326] [Closed Feature requests]
Hi Michel,

Can you please let me know whether MW4 can be executed via a third party software, in my case Voyager ? Let me explain:

I would like to execute automatically MW4 when certain criteria are met. In order to achieve that, I need to know what "argument" to use in order to execute MW4 (see attachment).

I hope you could help me.

Warm regards,

Aygen

<img width="548" alt="Capture d’écran 2023-09-15 à 17 09 15" src="https://github.com/mworion/MountWizzard4/assets/104280685/883ccaba-b95a-445d-8c64-e7326bdbcf7b">

**Comment by mworion** [comment]
Aygen,
actually MW4 could not be run in headless mode e.g. using a command line interface. There are some functions, which could be accessed via TCP, but very limited ones. 
Could you explain, which type of function you would like to run in case ?
Michel

**Comment by rkas12** [comment]
Hi michel,

Thank you for your reply, greatly appreciated. Meanwhile, I did manage to
find a "workaround". But for your information : the idea was to use a
script to automatically launch MW4, for example, once some criteria are
met.

Best regards,

Aygen

**Reply by mworion** [reply]
Hi Aygen,

could you explain the workaround also for others?
Still, you could start MW4 remotely and also stop it. question is which function you expect to work.

Michel
